src/main/scala/chisel3/util/Bitwise.scala

```diff
@@ -24,13 +26,27 @@ object FillInterleaved {
     *
     * Output data-equivalent to in(size(in)-1) (n times) ## ... ## in(1) (n times) ## in(0) (n times)
     */
-  def apply(n: Int, in: UInt): UInt = apply(n, in.asBools)
+  def apply(n: Int, in: UInt): UInt = macro SourceInfoTransform.nInArg
+
+  def do_apply(n: Int, in: UInt)(implicit sourceInfo: SourceInfo, compileOptions: CompileOptions): UInt =
```

Comment:
```suggestion
  /** @group SourceInfoTransformMacro */
  def do_apply(n: Int, in: UInt)(implicit sourceInfo: SourceInfo, compileOptions: CompileOptions): UInt =
```
This grouping helps a lot in the ScalaDoc, eg. https://www.chisel-lang.org/api/latest/chisel3/UInt.html

Reply:
We should add this to all new `do_` methods.

Reply:
do I need to create the group first for the `Fill` thing? I think I need to do more than just adding the `@group` right, I need to copy all the "here be dragons" stuff ...

Reply:
figured it out. fixed.